Actions/Helpers/AL-Go-Helper.ps1

```diff
@@ -1,1113 +1,1119 @@
-                do {
-                    Start-Sleep -Seconds 10
-                    $baseApp = Get-BcPublishedApps -bcAuthContext $authContext -environment $environmentName | Where-Object { $_.Name -eq "Base Application" }
-                } while (!($baseApp))
-                $baseapp | Out-Host
-            }
-            
-            $artifact = Get-BCArtifactUrl `
-                -country $countryCode `
-                -version $baseApp.Version `
-                -select Closest
-            
-            if ($artifact) {
-                Write-Host "Using Artifacts: $artifact"
-            }
-            else {
-                throw "No artifacts available"
-            }
-
-            $runAlPipelineParams += @{
-                "artifact" = $artifact
-                "bcAuthContext" = $authContext
-                "environment" = $environmentName
-                "containerName" = "bcServerFilesOnly"
-                "updateLaunchJson" = "Cloud Sandbox ($environmentName)"
-            }
-        }
-        
-        Run-AlPipeline @runAlPipelineParams `
-            -pipelinename $workflowName `
-            -imageName "" `
-            -memoryLimit $repo.memoryLimit `
-            -baseFolder $baseFolder `
-            -licenseFile $LicenseFileUrl `
-            -installApps $installApps `
-            -installTestApps $installTestApps `
-            -appFolders $repo.appFolders `
-            -testFolders $repo.testFolders `
-            -testResultsFile $testResultsFile `
-            -testResultsFormat 'JUnit' `
-            -installTestRunner:$repo.installTestRunner `
-            -installTestFramework:$repo.installTestFramework `
-            -installTestLibraries:$repo.installTestLibraries `
-            -installPerformanceToolkit:$repo.installPerformanceToolkit `
-            -enableCodeCop:$repo.enableCodeCop `
-            -enableAppSourceCop:$repo.enableAppSourceCop `
-            -enablePerTenantExtensionCop:$repo.enablePerTenantExtensionCop `
-            -enableUICop:$repo.enableUICop `
-            -azureDevOps:($caller -eq 'AzureDevOps') `
-            -gitLab:($caller -eq 'GitLab') `
-            -gitHubActions:($caller -eq 'GitHubActions') `
-            -failOn 'error' `
-            -AppSourceCopMandatoryAffixes $repo.appSourceCopMandatoryAffixes `
-            -AppSourceCopSupportedCountries @() `
-            -doNotRunTests `
-            -useDevEndpoint `
-            -keepContainer
-    }
-    finally {
-        # Cleanup
-        try {
-            Remove-Module BcContainerHelper
-            Remove-Item $bcContainerHelperPath -Recurse
-        }
-        catch {}
-    }
-}
-
-function ConvertTo-HashTable() {
-    [CmdletBinding()]
-    Param(
-        [parameter(ValueFromPipeline)]
-        [PSCustomObject] $object
-    )
-    $ht = @{}
-    if ($object) {
-        $object.PSObject.Properties | Foreach { $ht[$_.Name] = $_.Value }
-    }
-    $ht
-}
+Param(
```

Comment:
Moving this will cause all existing repositories to no longer be able to update AL Go System files.